cmd/main.go

```diff
@@ -1,3 +1,5 @@
+//go:build exclude_from_coverage
+
```

Comment:
This seems like an unconventional use of go build flags - usually coverage exclusion is achieved by specifying it in the Codacy exclusions, or by stripping those files from the coverage profile data directly. I also don't see any reference to this build flag anywhere else in the PR. Why do you want to exclude this file anyway?

Reply:
I've removed the flag, as it's not necessary to explicitly exclude these dir.